expression/aggregation.go

```diff
@@ -166,11 +238,25 @@ func (sf *sumFunction) Update(row []types.Datum, groupKey []byte, ctx context.Co
 	return sf.updateSum(row, groupKey, ctx)
 }
 
+func (sf *sumFunction) StreamUpdate(row []types.Datum, groupKey []byte, ectx context.Context) (bool, error) {
+	return sf.streamUpdateSum(row, groupKey, ectx)
+}
+
 // GetGroupResult implements AggregationFunction interface.
 func (sf *sumFunction) GetGroupResult(groupKey []byte) (d types.Datum) {
 	return sf.getContext(groupKey).Value
 }
 
+func (sf *sumFunction) GetStreamResult() (d types.Datum) {
+	defer func() {
+		sf.resultCtx = sf.streamCtx
```

Comment:
you mean change it to 
`d = sf.resultCtx.Value
 sf.resultCtx = sf.streamCtx
return
`
like this ?